molecule_openstack/driver.py

```diff
@@ -31,7 +31,7 @@ class Openstack(Driver):
     The class responsible for managing `OpenStack`_ instances.  `OpenStack`_
     is `not` the default driver used in Molecule.
 
-    Molecule leverages Ansible's `openstack_module`_, by mapping variables
+    Molecule leverages Ansiblessh_connection_options's `openstack_module`_, by mapping variables
```

Comment:
this bit looks weird

Reply:
yes it does... thinking it's a typo let me go see if I can find this code and ammend